benchmarks/benchmarks/stats_sampling.py

```diff
@@ -181,6 +186,35 @@ def time_pinv_rvs(self, dist):
         self.rng.rvs(100000)
 
 
+class NaiveRatioUniforms(Benchmark):
+
+    param_names = ['dist']
+
+    params = [allcontdists]
+
+    def setup(self, dist):
+        self.urng = np.random.default_rng(0xb235b58c1f616c59c18d8568f77d44d1)
+        # to run the benchmark, the bounding rectangle must be specified
+        # in the dist object
+        if hasattr(dist, 'nrou_bounds'):
+            u_min, u_max, v_max = dist.nrou_bounds()
+            self.rng = stats.NaiveRatioUniforms(
+                dist, random_state=self.urng, u_min=u_min, u_max=u_max,
+                v_max=v_max
+            )
+        else:
+            raise NotImplementedError(f"no rectangle specified for {dist}")
+
+    def time_nrou_setup(self, dist):
+        u_min, u_max, v_max = dist.nrou_bounds
```

Comment:
this needs to be `dist.nrou_bounds()`, otherwise an error is raised when running the benchmarks

Reply:
also, if you decide to keep `u_min`, `u_max`, and `v_max` optional, we can also check the performance of setup when bounding rectangles are not available.